woocommerce/woocommerce-ios
# REST API: UI for the custom WPCom email login screen of the Jetpack Setup flow

Part of #8918 


## Description

This PR adds the UI for the custom WPCom email login screen of the Jetpack setup flow. Only local validation for the email field is available for now, integration will be added in a future PR.

Reference for the flow: [pe5sF9-1c9-p2]
Figma file: [ERGtCFLyiC9CFpHsw1XIid-fi-1516%3A41678]

## Testing instructions

Prerequisite: Make sure that you have access to a site with WooCommerce and no Jetpack.
- Log out of the app or skip onboarding if needed.
- Tap "Enter your site address" on the prologue screen and proceed with the address of your test site.
- Log in with the credentials of an admin account.
- When the login completes, tap the Jetback benefit banner at the bottom of the Dashboard screen.
- Tap Install Jetpack on the benefit modal.
- The WPCom email login screen should be displayed. The Install Jetpack button on this screen should be enabled only when you type a valid email address. The screen's title, subtitle, and CTA should mention "Install Jetpack".
- Log out of the app and log in again to your test site with another account (either admin or shop manager).
- Install, activate and connect Jetpack for your test site using your main account on WP Admin.
- On the app, tap the Jetpack benefit banner and Install Jetpack on the modal.
- The WPCom email login screen should be displayed with the title, subtitle, and CTA mentioning "Connect Jetpack".

## Screenshots

| Needs installation | Connection only |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/221568969-c0a71923-7d8b-4de0-8596-f6792cdfb320.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/221575671-b4920811-615b-401b-aa29-4a23364e0d5c.png" width=320 />

---
